holland/legacy/config.py

```diff
@@ -25,8 +25,14 @@ def extract_params(section):
 def get_config(**kwargs):
     global config
     global spec
+
+    holland_conf = '/etc/holland/holland.conf'
+    currplatform = sys.platform
```

Comment:
Tiny nitpick, but maybe `curr_platform` or better, `current_platform` to match the other variables?